flink-python/pyflink/fn_execution/coders.py

```diff
@@ -414,6 +415,25 @@ def __repr__(self):
         return 'ArrowCoder[%s]' % self._schema
 
 
+class CustomLengthPrefixCoder(LengthPrefixCoder):
+    """
+    CustomLengthPrefixCoder will replace LengthPrefixCoder in Beam for performance optimization.
```

Comment:
Coder which doesn't prefix the length of the encoded object as the length prefix will be handled by the inner coder.